roles/setup/tasks/setup-container.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Enable linger for {{ receptor_user }}
+  become: true
+  ansible.builtin.command:
+    cmd: "loginctl enable-linger {{ receptor_user }}"
+  changed_when: false
+
+- name: Get UID of {{ receptor_user }}
```

Comment:
you can get user facts with `ansible.builtin.user`:

```
- name: Gather user info
  ansible.builtin.user:
    name: "{{ receptor_user }}"
    state: present
  register: receptor_user_register
```

We should avoid `shell/command` if you can perform the same task with a builtin module

Reply:
Great note, I updated to capture this